jcl/src/java.base/share/classes/java/lang/ref/Reference.java

```diff
@@ -15,6 +15,9 @@
 import sun.misc.SharedSecrets;
 /*[ENDIF]*/
 /*[ENDIF]*/
+/*[IF Java11]*/
+java.lang.ref.Finalizer;
```

Comment:
This isn't a valid statement and won't compile. It its supposed to be an import, no import is required since Finalizer is in the same package.

Reply:
have removed it